### PR TITLE
Add some seeds

### PR DESCRIPTION
**Pull request by sebastiantoh**
```
bundle exec rails add_growth_and_school_challenge
bundle exec rails add_productivity_challenge
```

### CODE REVIEW

File: backend/lib/tasks/20211105103312_add_growth_and_school_challenge.rake [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+                                      schedule: 'Every day for 1 month',
+                                      creator_id: user_odyssey.id,
+                                      color: 2,
+                                      original_creator: 'Matt Valentine',
+                                      link_to_reference: 'https://www.goalcast.com/30-day-growth-challenge/')
+    Map.create!(land: 0, challenge_id: challenge_growth.id)
+
+    Task.create!(challenge_id: challenge_growth.id,
+                name: 'Day 1',
+                description: 'Last last year behind and commit to no expectations. This could be something as simple as taking a short walk on the beach and allowing yourself to let the past rest behind you.',
```
Comment (jialin7878) [inline backend/lib/tasks/20211105103312_add_growth_and_school_challenge.rake:31]: Last last year?

File: backend/lib/tasks/20211105103312_add_growth_and_school_challenge.rake [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+                 index: 2)
+
+    Task.create!(challenge_id: challenge_growth.id,
+                 name: 'Day 4',
+                 description: 'List important assets you need to accomplish those goals. Chances are, you don’t have everything you need to accomplish your goals. Make a list and figure how you’ll acquire the assets you need to do that (such as skills, new contacts, etc.).',
+                 index: 3)
+
+    Task.create!(challenge_id: challenge_growth.id,
+                 name: 'Day 5',
+                 description: 'Make a vision board. I know, sounds hoaky – but it’s not. Don’t pass this one up! (Even if it takes more than a day.)',
```
Comment (jialin7878) [inline backend/lib/tasks/20211105103312_add_growth_and_school_challenge.rake:51]: hoaky not a word(?)

File: backend/lib/tasks/20211105103312_add_growth_and_school_challenge.rake [hunk 3]
```diff
@@ -0,0 +1,332 @@
...
+                 index: 8)
+
+    Task.create!(challenge_id: challenge_growth.id,
+                 name: 'Day 10',
+                 description: 'Adopt a nightly ritual. Like the morning routine, but this is all about optimizing your sleep and well-being by placing you in the right physical and mental state of being before retiring for the day.',
+                 index: 9)
+
+    Task.create!(challenge_id: challenge_growth.id,
+                 name: 'Day 11',
+                 description: 'Minimize or get rid of T.V. I know, you can’t do without Game of Thrones. That’s okay, as long as you reduce your T.V. consumption down to just a few hours a week you’re good.',
```
Comment (jialin7878) [inline backend/lib/tasks/20211105103312_add_growth_and_school_challenge.rake:81]: nit/ can ignore: shld we update this to TV or YouTube/ Netflix/ streaming services? tv abit outdated (so is game of thrones haha)

File: backend/lib/tasks/20211105103312_add_growth_and_school_challenge.rake [hunk 4]
```diff
@@ -0,0 +1,332 @@
...
+                 index: 14)
+
+    Task.create!(challenge_id: challenge_growth.id,
+                 name: 'Day 16',
+                 description: 'Try a brain game. These are great for optimizing your cognitive functions.',
+                 index: 15)
+
+    Task.create!(challenge_id: challenge_growth.id,
+                 name: 'Day 17',
+                 description: 'Adopt a mindfulness practice.',
```
Comment (jialin7878) [inline backend/lib/tasks/20211105103312_add_growth_and_school_challenge.rake:111]: vague

File: backend/lib/tasks/20211105103312_add_growth_and_school_challenge.rake [hunk 5]
```diff
@@ -0,0 +1,332 @@
...
+                 index: 19)
+
+    Task.create!(challenge_id: challenge_growth.id,
+                 name: 'Day 21',
+                 description: 'Today is a free day. Use this time to have a break, or make up for missed objectives!',
+                 index: 20)
+
+    Task.create!(challenge_id: challenge_growth.id,
+                 name: 'Day 22',
+                 description: 'Start reading. Depending on your profession, this can be anything that brings you value. Same goes for the next point.',
```
Comment (jialin7878) [inline backend/lib/tasks/20211105103312_add_growth_and_school_challenge.rake:136]: same goes for `the next task` ? won't be able to see next point in same page

File: backend/lib/tasks/20211105103312_add_growth_and_school_challenge.rake [hunk 6]
```diff
@@ -0,0 +1,332 @@
...
+
+    Task.create!(challenge_id: challenge_growth.id,
+                 name: 'Day 26',
+                 description: 'Reach out to one person that can move your career forward. This is all about building professional connections. Most of us have a hard time bringing ourselves to make that initial contact, let alone build entire new relationships with people. This is about taking that initial step, everything else tends to happen naturally from there, because we’re motivated to continue.',
+                 index: 25)
+
+    Task.create!(challenge_id: challenge_growth.id,
+                 name: 'Day 27',
+                 description: 'Perform an act of kindness. You’ll notice this doesn’t seem to fit with the theme of this week. However, I’ve found that most of us have lost touch with (or never knew) the power of helping others without expecting anything in return. This knowledge, and the understanding that you always have the ability to exercise it, doesn’t just make you happier and more fulfilled, it can serve you in your career in a big way. Such acts of kindness are the perfect way to start off a long and beautiful business relationship or personal connection.',
```
Comment (jialin7878) [inline backend/lib/tasks/20211105103312_add_growth_and_school_challenge.rake:161]: slightly rambly

File: backend/lib/tasks/20211105115844_add_productivity_challenge.rake [hunk 7]
```diff
@@ -0,0 +1,138 @@
+# frozen_string_literal: true
+# rubocop:disable all
+
+# task_list is a list of the form { name: string, description: string }[].
+# If the name parameter is omitted, then the name of the task will simply be "Day x"
+def create_tasks(task_list, challenge_id)
```
Comment (jialin7878) [inline backend/lib/tasks/20211105115844_add_productivity_challenge.rake:6]: nicee

File: backend/lib/tasks/20211105115844_add_productivity_challenge.rake [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+      },
+      {
+        description: "Silence your phone"
+      },
+      {
+        description: "Create a workspace you love"
+      },
+
+      {
+        description: "Find an ergonomic chair"
```
Comment (jialin7878) [inline backend/lib/tasks/20211105115844_add_productivity_challenge.rake:56]: just find one(?)

File: backend/lib/tasks/20211105115844_add_productivity_challenge.rake [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+      },
+      {
+        description: "De-clutter your workspace"
+      },
+      {
+        description: "Play upbeat music to boost energy and simulate alertness"
+      },
+      {
+        description: "Drink enough water"
```
Comment (jialin7878) [inline backend/lib/tasks/20211105115844_add_productivity_challenge.rake:65]: nit: change to 'more water'? or some measure of how much is enough

File: backend/lib/tasks/20211105115844_add_productivity_challenge.rake [hunk 10]
```diff
@@ -0,0 +1,138 @@
...
+        description: "Learn to say No"
+      },
+      {
+        description: "Designate a break for social media"
+      },
+      {
+        description: "Use your commute time for minor tasks such as checking and answering emails, or catching up on readings"
+      },
+      {
+        description: "Exercise daily"
```
Comment (jialin7878) [inline backend/lib/tasks/20211105115844_add_productivity_challenge.rake:122]: exercise 'today'?

File: backend/lib/tasks/20211105115844_add_productivity_challenge.rake [hunk 11]
```diff
@@ -0,0 +1,138 @@
...
+      },
+      {
+        description: "Make a weekly list of your achievements"
+      },
+      {
+        description: "Reward yourself"
+      },
+      {
+        description: "Sleep at least 8 hours every night"
```
Comment (jialin7878) [inline backend/lib/tasks/20211105115844_add_productivity_challenge.rake:131]: remove 'every night'?

**Review by jialin7878 — APPROVED** [review]
thanks a lot!! I left comments but they're all the fault of the og creator not making sense, feel free to ignore as you see fit!

**Comment by jialin7878** [comment]
oh and also, 'Challenge' in the name seems abit redundant to always include, and the other challenges don't have 'Challenge' in the name also..could it be something like 'Becoming more productive' instead of 'Productivity Challenge'?

**Comment by jialin7878** [comment]
Side observation all the productivity challenges are a completely different task each day :0 which def goes against our forming a habit thing (but issok dunnid change them!)

File: backend/lib/tasks/20211105115844_add_productivity_challenge.rake (hunk 8 above)
Comment (sebastiantoh) [reply backend/lib/tasks/20211105115844_add_productivity_challenge.rake:56]: changed it to `Shop for an ergonomic chair`. not sure if it's better haha

**Comment by sebastiantoh** [comment]
ty for the very thorough check! appreciate it

File: backend/lib/tasks/20211105115844_add_productivity_challenge.rake (hunk 8 above)
Comment (jialin7878) [reply backend/lib/tasks/20211105115844_add_productivity_challenge.rake]: Hahaha hmm maybe 'Read up about ergonomic principles and apply what you learnt to improve your own workspace' (?) Ergonomic chairs are rlly expensive

**Comment by ianyong** [comment]
> bundle exec rails add_growth_and_school_challenge
> bundle exec rails add_productivity_challenge

Ran on production!

**Pull request merged**